cmake/external/ROOT.cmake

```diff
@@ -11,8 +11,18 @@ if(APPLE)
      "${DETECTED_OS_VERS}" MATCHES "^osx-12" OR
      "${DETECTED_OS_VERS}" MATCHES "^osx-11.6" OR
      "${DETECTED_OS_VERS}" MATCHES "^osx-11.7")
+    # Test if Xcode is installed via `xcodebuild -version`.
+    message(STATUS "##### Checking if XCODE is installed")
+    execute_process(COMMAND bash "-c" "xcodebuild -version" ERROR_VARIABLE XCODE_ERROR)
+    if("${XCODE_ERROR}" STREQUAL "")
+      message(STATUS "##### XCODE is installed")
+    else()
+      message(FATAL_ERROR "##### XCODE is not installed correctly. Please install XCODE (and then the command line tools). Consult issue #352 and the documentation")
```

Comment:
Could you please add the full link to the issue in parenthesis?

Reply:
See https://github.com/BioDynaMo/biodynamo/pull/353/commits/e8d13066dc7b04663d8705c2f795b7c70b4fd7c8